benchmarks/auth/knative_yamls/auth-python.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: auth-python
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+        - image: docker.io/vhiveease/auth-python:latest
+          imagePullPolicy: Always
+          # args: ["--knative", "--verbose", "--out", "s3://vhive-corral-bench/word_count/vhive-ubench/out"]
```

Comment:
I suppose that it works so I think we should remove all the commented lines or not?